src/libraries/Microsoft.Extensions.Logging/tests/Microsoft.Extensions.Logging.Generators.Tests/Baselines/TestWithDynamicLogLevel.generated.txt

```diff
@@ -0,0 +1,57 @@
+// <auto-generated/>
+#nullable enable
+
+namespace Microsoft.Extensions.Logging.Generators.Tests.TestClasses
+{
+    partial class TestWithDynamicLogLevel 
+    {
+        [global::System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.Extensions.Logging.Generators", "6.0.0.0")]
+        private readonly struct __M9Struct : global::System.Collections.Generic.IReadOnlyList<global::System.Collections.Generic.KeyValuePair<string, object?>>
+        {
+
+            public override string ToString()
+            {
+
+                return $"M9";
+            }
+
+            public static string Format(__M9Struct state, global::System.Exception? ex) => state.ToString();
+
+            public int Count => 1;
+
+            public global::System.Collections.Generic.KeyValuePair<string, object?> this[int index]
+            {
+                get => index switch
+                {
+                    0 => new global::System.Collections.Generic.KeyValuePair<string, object?>("{OriginalFormat}", "M9"),
+
+                    _ => throw new global::System.IndexOutOfRangeException(nameof(index)),  // return the same exception LoggerMessage.Define returns in this case
+                };
+            }
+
+            public global::System.Collections.Generic.IEnumerator<global::System.Collections.Generic.KeyValuePair<string, object?>> GetEnumerator()
+            {
+                for (int i = 0; i < 1; i++)
+                {
+                    yield return this[i];
+                }
+            }
+
+            global::System.Collections.IEnumerator global::System.Collections.IEnumerable.GetEnumerator() => GetEnumerator();
+        }
+
+        [global::System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.Extensions.Logging.Generators", "6.0.0.0")]
+        public static partial void M9(Microsoft.Extensions.Logging.LogLevel level, Microsoft.Extensions.Logging.ILogger logger)
```

Comment:
Do these type references need `global::` like we do elsewhere?

```C#
global::Microsoft.Extensions.Logging.LogLevel level, global::Microsoft.Extensions.Logging.ILogger logger
```

Reply:
`global::` is necessary for cases when generated code gets the wrong symbol depending on the namespace it is being generated into.
To avoid this kind of potential risk, generated code generally adds a `global::` in front of all symbols.

I think it is fine to keep.

Reply:
My comment was that these **don't** have `global::`

Reply:
oh sorry didn't realize

Reply:
Was this addressed?

Reply:
will be in my next commit